raiden_contracts/tests/test_one_to_n.py

```diff
@@ -119,17 +121,57 @@ def test_claim(
         ).call({"from": A})
 
 
+def test_claim_by_unregistered_service(
+    one_to_n_contract: Contract,
+    deposit_to_udc: Callable,
+    get_accounts: Callable,
+    get_private_key: Callable,
+    web3: Web3,
+) -> None:
+    """OneToN contract should not work for an unregistered service provider."""
+    (A, B) = get_accounts(2)
+    deposit_to_udc(A, 30)
+
+    # happy case
```

Comment:
why is this the happy case?

Reply:
No, I copied and pasted the happiness but left the unhappy case only.